routes/api.js

```diff
@@ -0,0 +1,26 @@
+var express = require('express');
+var router = express.Router();
+
+var bodyParser = require('body-parser');
+
+router.post('/api/user-access', function(req, res) {
```

Comment:
I think you could put `router.post('/user-access', ...`
and then when you mount these route handlers in `app.js` you can have it prepend '/api' to the beginning of all of the routes in this module by doing something like this:
```
api = require('./routes/api')
app.use('/api', api)
````
it looks like this is what's being done on lines 9 and 27